MDHelpers/MDLog.cs

```diff
@@ -230,8 +230,9 @@ private static void LogToFile(string Message)
     // Creates our time-stamped log file and opens it for writing
     private static void InitLogFile()
     {
-        FullLogFilePath = LOG_DIR + DateTime.Now.ToString("yyyy.MM.dd-HH.mm.ss") + EXT_LOG;
-        if (CreateLogDirectoryIfNotExists(LOG_DIR))
+        String logDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "\\logs\\";
```

Comment:
Sorry about that didn't mean for this code to be checked in, I couldn't figure out where this user:// directory was so I added my own. But will add the virtual

Reply:
I actually have to scrap this, because the game instance starts the logger right as it starts up in ready before it is added to the tree. As such I will just leave it the old way and add a comment about where to find the logs.

Reply:
The game instance could pass the directory into the Initialize method